pkg/core/config/writer.go

```diff
@@ -201,3 +237,32 @@ func (wc *WriterConfig) DefaultTraceEndpointPath() string {
 	}
 	return "/v1/trace"
 }
+
+// SplunkConfig configures the writer specifically writing to Splunk.
+type SplunkConfig struct {
+	// Enable logging to a Splunk Enterprise instance
+	Enabled bool `yaml:"enabled"`
+	// url to Splunk HTTP Event Collector
+	URL string `yaml:"url"`
+	// Splunk HTTP Event Collector token
+	Token string `yaml:"token"`
+	// Splunk source field value, description of the source of the event
+	Source string `yaml:"source"`
```

Comment:
@atoulme should this field `source` be required or is it also optional?

Reply:
It can be optional. The user can set a default source.